molecule/src/molecule_ionize.cpp

```diff
@@ -444,433 +419,423 @@ void MoleculePkaModel::_loadSimplePkaModel()
     _model.simple_model_ready = true;
 }
 
-void MoleculePkaModel::_loadAdvancedPkaModel()
+namespace  // only for data
 {
-    enum PkaType
-    {
-        PKA_ACID,
-        PKA_BASIC
-    };
-
-    struct _PkaDef
+    struct PkaDef
```

Comment:
It also makes sense to reuse this type and refactor this:
```
    static _PkaDef simple_pka_model[] = {...
```
inside `MoleculePkaModel::_loadSimplePkaModel()`